source/components/composing-components.md

```diff
@@ -0,0 +1,143 @@
+Components really shine when you use them to their full potential, which is when you compose them.
+Take for example the `<ul>` element, and the fact that only `<li>` elements are appropriate as children.
+If we want the same type of behavior, then we have to compose our components.
+
+Just like we compose regular HTML elements, we can do the same with components.
+
+```app/templates/application.hbs
+{{#user-list sort-by='name' as |user|}}
```

Comment:
@knownasilya This may be a bad example, but do you think it would look strange if you bound data down through multiple components?

```handlebars
{{#user-list sort-by='name' as |user|}}
  {{sort-controls sort-by=sortBy}}
{{/user-list}}
```